maintainers/scripts/luarocks-packages.csv

```diff
@@ -3,10 +3,17 @@ ansicolors,,,,,
 argparse,,,,,
 basexx,,,,,
 binaryheap,,,,,vcunat
+bit32,,,,lua5_1,lblasc
+busted,,,,,
+cjson,lua-cjson,,,,
+compat53,,,,,vcunat
+coxpcall,,,1.17.0-1,,
+cqueues,,,,,vcunat
```

Comment:
we had some problems with cqueues previously (they had one rockspeck per lua version) but seems like you solved them in the overrides. Cool